packages/@expo/cli/src/lint/index.ts

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env node
+
+import chalk from 'chalk';
+
+import { Command } from '../../bin/cli';
```

Comment:
Definitely not required, usually I add `import { type Command } ...` or `import type { Command } ...` to help the bundler skip over this import when transpiling. We use swc with taskr here, so you won't really notice any effects (but it could very well be in other Expo projects) -- it might even help reduce potential cyclic dependency warnings.

```suggestion
import { type Command } from '../../bin/cli';
```